claim_ai/apps.py

```diff
@@ -7,22 +7,33 @@
 
 logger = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
+
 MODULE_NAME = 'claim_ai'
 
 DEFAULT_CONFIG = {
     'authentication': [],
     'claim_response_url': 'http://localhost:8000/claim_ai/ClaimResponse',
     'ai_model_file': "",
```

Comment:
you can add the default ai_model_file. We need to be able to replace this trough docker and .env and to place the files outside docker container.

Reply:
I added 'joblib_Voting_Model.pkl' as default value